src/cogs/trivia.py

```diff
@@ -49,6 +51,38 @@ def tally_scores(
                 (loser, ))
         self.db.commit()
 
+    @commands.command(usage="somechannel")
+    @commands.has_permissions(manage_roles=True)
+    async def triviachan(self, ctx, channel):
```

Comment:
This formatting is a little weird, mind cleaning it up a little? ypaf should be installed as a dev package with pipenv.